versionist.conf.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const execSync = require('child_process').execSync;
+const yaml = require('yaml');
+const fs = require('fs');
+const regex = /(docker:\/\/ghcr.io\/balena-io\/deploy-to-balena-action):(.*)/i;
```

Comment:
Since we don't care about the second matching group we can just remove it:
```suggestion
const regex = /(docker:\/\/ghcr.io\/balena-io\/deploy-to-balena-action):.*/i;
```